packages/core/package.json

```diff
@@ -100,7 +99,7 @@
     "eslint-plugin-react-native": "^3.8.1",
     "expo": "^52.0.0",
     "expo-module-scripts": "3.1.0",
-    "jest": "^29.6.2",
+    "jest": "^29.7.0",
```

Comment:
L: Were there some issues with the older jest and the new JS SDK? We can keep it, I'm just curious.

Reply:
There were, but not anymore, so I am returning to the older version